amplify.yml

```diff
@@ -15,3 +15,6 @@ frontend:
       - '**/*'
   cache:
     paths: []
+
+# Please note that beyond this YML file, we also have redirects configured in the AWS Console.
```

Comment:
these should come from the amplify terraform, but it seems I forgot to add this yet 😬 
let's f2f sync about this